src/wallet/scriptpubkeyman.cpp

```diff
@@ -2644,17 +2644,27 @@ const WalletDescriptor DescriptorScriptPubKeyMan::GetWalletDescriptor() const
 }
 
 const std::unordered_set<CScript, SaltedSipHasher> DescriptorScriptPubKeyMan::GetScriptPubKeys() const
+{
+    return GetScriptPubKeys(0);
+}
+
+const std::unordered_set<CScript, SaltedSipHasher> DescriptorScriptPubKeyMan::GetScriptPubKeys(int32_t minimum_index) const
```

Comment:
`const` is still there

Reply:
> > Thanks, done with a preparatory extra refactoring commit which changes the ScriptPubKeyMan::GetScriptPubKeys() interface (which imposes this on us).
> 
> `const` is still there

See https://github.com/bitcoin/bitcoin/pull/25957#issuecomment-1277486053:
> removed the commit that got rid of the const qualifier from GetScriptPubKeys, since it was seen as controversial and is not really related to this PR (can be done as a follow-up).